src/ace.F90

```diff
@@ -1089,6 +1097,8 @@ recursive subroutine get_energy_dist(aedist, law, LDIS, IDAT, awr, Q_value)
       allocate(KalbachMann :: aedist)
     elseif (law == 61) then
       allocate(CorrelatedAngleEnergy :: aedist)
+    elseif (law == 66) then
```

Comment:
So for now we will keep the law numbers from ENDF/ACE, or are you planning on changing that in one of the subsequent PRs which move towards H5 data?